fiftyone/core/collections.py

```
@@ -11538,40 +11556,270 @@ async def _async_aggregate(self, aggregations, maxTimeMS=None):
         # Placeholder to store results
         results = [None] * len(aggregations)
 
-        idx_map = {}
-        pipelines = []
-
         if facet_aggs:
             # Build facet-able pipelines
             compiled_facet_aggs, facet_pipelines, hints = self._build_facets(
```

Ruff (0.15.9)

[warning] Unpacked variable `hints` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)